doc/quickref.xml

```diff
@@ -397,14 +397,6 @@
     <simplelist type="vert" columns="1">
      <member><link linkend="geometry.reference.algorithms.densify">densify</link></member>
     </simplelist>
-    <bridgehead renderas="sect3">Discrete_Frechet_Distance</bridgehead>
-    <simplelist type="vert" columns="1">
-     <member><link linkend="geometry.reference.algorithms.discrete_frechet_distance">discrete_frechet_distance</link></member>
-    </simplelist>
-    <bridgehead renderas="sect3">Discrete_Hausdorff_Distance</bridgehead>
-    <simplelist type="vert" columns="1">
-     <member><link linkend="geometry.reference.algorithms.discrete_hausdorff_distance">discrete_hausdorff_distance</link></member>
-    </simplelist>
```

Comment:
You should leave both algorithms but use only one title, like here:
https://github.com/boostorg/geometry/blob/develop/doc/quickref.xml#L326